3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -0,0 +1,227 @@
+#ifndef OPENVX_HAL_HPP_INCLUDED
+#define OPENVX_HAL_HPP_INCLUDED
+
+#include <cstdio>
+#include "opencv2/core/hal/interface.h"
+#include "VX/vx.h"
+#include "VX/vxu.h"
+#include <string>
+#include <iostream>
+
+//===================================
+// utility
+// ...
+
+#include <chrono>
```

Comment:
std::chrono::steady_clock requires [C++11](http://en.cppreference.com/w/cpp/chrono/steady_clock), which is not used in OpenCV by default. So this feature should be optional.